docs/astro.config.mjs

```diff
@@ -8,4 +9,12 @@ export default {
     // Needed for Algolia search component
     '@astrojs/renderer-react',
   ],
-};
+  vite: {
+    resolve: {
+      alias: {
+        '~': './src',
+        'components': './src/components',
```

Comment:
This is a soft suggestion: aliasing directories was required in Snowpack, but is not in Vite. I think we’d prefer people use “real” paths as the default. But I’d be in favor of moving these to a guide on aliasing for people who want to opt in.

Reply:
Only coming from past experience where many people would set up aliasing, only to find them confusing to work with

Reply:
I think the purpose here is just to test that the feature works, since we do support it.

Reply:
1. This aliasing has been updated in https://github.com/snowpackjs/astro/pull/1738/commits/b37e2dd8b697181f28c42754c9e69bc29de7b59e
2. This aliasing may be superseded by https://github.com/snowpackjs/astro/pull/1747 to provide consistent aliasing to both Astro and the IDE. IMHO, aliasing that is not also picked up by the editor is confusing to work with.